src/app/assets/javascripts/angular/app.js

```diff
@@ -15,28 +15,52 @@
         controller: 'LoginController'
       })
       .when('/main', {
-        templateUrl: 'card.html',
-        controller: 'CardController'
+        templateUrl: 'timings.html',
+        controller: 'TimingsController'
       })
       .when('/location', {
         templateUrl: 'location.html',
         controller: 'LocationController'
       })
+      .when('/all', {
+        templateUrl: 'all.html',
+        controller: 'LocationController'
+      })
+      .when('/leaderboards', {
+        templateUrl: 'leaderboards.html',
+        controller: 'LeaderboardsController'
+      });
   }]);
 
-  app.directive('nbTimings', function() {
+  app
+  .directive('nbHeader', function() {
     return {
-      templateUrl: 'timings.html'
+      templateUrl: 'header.html'
     };
-  });
-  app.directive('nbReport', function() {
+  })
+  .directive('nbTimings', function() {
+    return {
+      templateUrl: 'timings-list.html'
+    };
+  })
+  .directive('nbReport', function() {
     return {
       templateUrl: 'report.html'
     };
-  });
-  app.directive('nbLocation', function() {
+  })
+  .directive('nbLocation', function() {
     return {
       templateUrl: 'stop-selection.html'
     };
+  })
+  .directive('nbAllStops', function() {
+    return {
+      templateUrl: 'all-stops.html'
+    };
+  })
+  .directive('nbBoards', function() {
+    return {
+      templateUrl: 'leaderboard-table.html'
+    };
   });
 })();
```

Comment:
Potentially has conflict in this file when you merge this into master. The conflicts will be mostly about naming of controller and templates. Please check carefully :)